src/http/login.cpp

```diff
@@ -125,13 +125,20 @@ std::pair<status, json::value> tfs::http::handle_login(const json::object& body,
 		} while (result->next());
 	}
 
+	std::string serverIp = getString(ConfigManager::IP);
+	if (getBoolean(ConfigManager::ALLOW_OTC_PROXY)) {
+		serverIp = std::string("127.0.0.1");
+	} else if (getBoolean(ConfigManager::ALLOW_HAPROXY)) {
+		serverIp = getString(ConfigManager::STATUS_IP);
+	}
```

Comment:
I think this makes it clear that only one of them should be set to true.
In addition, since the PROXY protocol is not limited to a trusted source, it should be noted that TFS should not be available publicly directly. Which is probably obvious in the context of using a proxy system but probably TFS should start rejecting connections without the PROXY header.